readme.md

```diff
@@ -1,3 +1,5 @@
 Este será el readme de la app.
 
 Se me olvidó meter una segunda línea.
+
+Ahora incluyo información específica de la UI.
```

Comment:
yo pondria interfaz de usuario en vez de UI

Reply:
Fixed in #a09c6372dd872e7588469fa0a83aa58d6b106b07